tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -133,7 +146,8 @@ def _process_and_assert_no_configuration_changes(
         repo_trees: Mapping[str, Sequence[str]],
         frames: Sequence[Mapping[str, str | bool]],
         platform: str,
-    ) -> tuple[list[CodeMapping], list[str]]:
```

Comment:
We don't need to return `in_app_stacrace_rules` since we can test it within this function.
<img width="593" alt="image" src="https://github.com/user-attachments/assets/94eff3fc-d27a-44ed-b203-4c4445cc2c4d" />